tools/lint/lint.py

```diff
@@ -903,6 +925,8 @@ def create_parser():
                         "using fnmatch, except that path separators are normalized.")
     parser.add_argument("--all", action="store_true", help="If no paths are passed, try to lint the whole "
                         "working directory, not just files that changed")
+    parser.add_argument("--github-checks-text-file", dest="github_checks_text_file",
```

Comment:
```suggestion
    parser.add_argument("--github-checks-text-file", type=ensure_text,
```

`dest` is redundant (the same as the default); `ensure_text` gives us a text string regardless of the Python version